pkg/gcptarget/gcptarget_test.go

```diff
@@ -118,6 +118,38 @@ var matchTests = []reviewTestData{
 		ancestryPath: "organizations/123454321/folders/1221214",
 		wantMatch:    true,
 	},
+	{
+		name: "organizations/** can match organizations/unknown",
+		match: map[string]interface{}{
+			"ancestries": []interface{}{"organizations/**"},
+		},
+		ancestryPath: "organizations/unknown/folders/1221214",
+		wantMatch:    true,
+	},
+	{
+		name: "organizations/* can NOT match organizations/unknown",
```

Comment:
Could you add a test for whether it matches `organizations/unknown` if the ancestryPath is `"organizations/unknown"` without any additional parts? (I think it _should_ but I could be wrong.)

Reply:
Added. Yes, it can match organizations/x, but not organizations/x/folders/y